README.md

```diff
@@ -1,80 +1,111 @@
 
-> You can install a preview version with limited functionality via `cargo install plow_cli`
-
-[Prebuilt binaries are coming soon!](https://github.com/field33/plow/issues/2)
+A new ontology editor is in the works, and will be released as a separate project.
 
 ## Basic usage
+
 ### Login with Plow
-The tooling currently expects you to be authenticated with the public plow registry ([open issue](https://github.com/field33/plow/issues/11)). Please sign in, creating an account [here](plow.pm) and [create a new User Token in your account settings](https://staging-registry.field33.com/home#user-tokens).
 
+The tooling currently expects you to be authenticated with the public plow registry ([open issue](https://github.com/field33/plow/issues/11)). Please sign in, creating an account [here](plow.pm) and [create a new api token in your account settings](https://staging-registry.field33.com/home#user-tokens).
 
-```shell
-plow login <YOUR_TOKEN>
+```sh
+plow login <api-token>
 ```
 
 ### Initialize workspace
-Create the directory you want to organize your *fields* (= ontologies) in.
-```shell
+
+Create the directory you want to organize your _fields_ (= ontologies) in.
+
+```sh
 # Create workspace directory
 mkdir example_workspace && cd example_workspace
 
-# Workaround: plow init currently expects a .ttl file to be present
-touch test.ttl
+# Crate your first field (.ttl file), or copy existing fields into the workspace
+plow init --field @example_namespace/example_name
 
-# Initializes the workspace in your directory
+# To initialize a workspace run
 plow init
 ```
 
+### Initialize a new _field_ (= ontology)
 
-### Initialize a new *field* (= ontology)
-To create a new *field* with all the necessary metadata run:
-```shell
-plow init --field @example_namespace/example_fieldname
+To create a new _field_ with all the necessary metadata run:
+
+```sh
+plow init --field @example_namespace/example_name
 ```
-This will create the relevant folder structure:
+
+When run under an initialized workspace, this will create the relevant folder structure in the `fields` directory:
+
 ```
 ├── Plow.toml
 └── fields
     └── @example_namespace
         └── example_fieldname.ttl
 ```
 
-### Submit an *field* to the registry
-To prepare for submitting a new *field* run the following command:
+If run outside of a workspace, it will create a new `.ttl` file in the current directory.
+
+```
+├── example_fieldname.ttl
+```
+
+Running `plow init` without the `--field` flag initializes a new workspace and if run after this, results would look like the most upper example.
+
+### Open a _field_ in protege
+
+If you'd like to open an edit a field in protege, you may use the following command:
+
+```sh
+# Continuing with the upper example
+plow protege example_fieldname.ttl
+```
+
+If you have protege installed in your system and if your field does not have parsing errors, this command will,
+
+- Resolve the dependencies in your field if there are some.
```

Comment:
> if there are some
May be `if any` is more grammatically correct? Not sure though as I'm not a native speaker.

Reply:
d9076d1